macro/ShipReco.py

```diff
@@ -77,6 +77,10 @@ def mem_monitor():
  options.geoFile = tmp.replace('_rec','')
 
 fgeo = ROOT.TFile.Open(options.geoFile)
+ROOT.gSystem.Load("libgenfit2")
+ROOT.gInterpreter.Declare('#include "TGeoMaterialInterface.h"')
+ROOT.gInterpreter.Declare('#include "FieldManager.h"')
+ROOT.gInterpreter.Declare('#include "MaterialEffects.h"')
```

Comment:
Can probably be removed now that the autoloading works correctly.

Reply:
Yes this can be removed.